static-analysis/autodispose-lint/src/test/kotlin/com/uber/autodispose/lint/AutoDisposeDetectorTest.kt

```diff
@@ -530,6 +530,57 @@ class AutoDisposeDetectorTest {
         .expectClean()
   }
 
+  @Test fun overrideCustomScopeWithoutAutoDispose() {
+    val properties = projectProperties()
+    properties.property(CUSTOM_SCOPE_KEY, "com.uber.autodispose.sample.ClassWithCustomScope")
+    properties.property(OVERRIDE_SCOPES, "true")
+    properties.to(AutoDisposeDetector.PROPERTY_FILE)
+
+    lint().files(rxJava2(), LIFECYCLE_OWNER, ACTIVITY, properties, kotlin("""
```

Comment:
nit - for readability, please put each of these args starting on separate lines

Reply:
3c42327dcf482d18b76ed02b31e491fcf282fa52

Have changed it for the two methods I added but the remaining ones have that. Can do another PR for those to clean up the formatting there. On a side note, should we think about a checkstyle to enforce conventions?

Reply:
cleaning up the rest separately is fine. There's an open issue for code style #154, but in general try to match what the rest of the project does too. My goal would be to use something like spotless